src/Command/ConfigDumperCommand.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\ServiceManager\Command;
+
+use InvalidArgumentException;
+use Laminas\ServiceManager\Exception;
+use Laminas\ServiceManager\Tool\ConfigDumper;
+use Laminas\ServiceManager\Tool\ConfigDumperInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function array_keys;
+use function assert;
+use function class_exists;
+use function dirname;
+use function file_exists;
+use function file_put_contents;
+use function is_array;
+use function is_string;
+use function is_writable;
+use function sprintf;
+
+final class ConfigDumperCommand extends Command
```

Comment:
`symfony/console` is in `require-dev`, yet we have a direct dependency to it under `src/` here :thinking:

Reply:
Yeah, commands are not meant to be used like services.
I did the same in `laminas-cache` as I do not require upstream projects to have `laminas-cli`.

The `ConfigProvider` does not inject anything `laminas-cli` related until `symfony/console` is available (which would be the case when either a `laminas-cli` or `symfony/console` is used.

Therefore, I've added it as an optional dependency - but might worth to add it to `suggest` 🤔 

The services (which are meant to be re-used) can still be used without `symfony/console` tho.


So I guess we have two options:
1. move `symfony/console` to `require`
2. add `laminas-cli` to `suggest` and mark the commands as `@internal` which implies it should not be instantiated from outside

WDYT?

Reply:
Since there was no further feedback, I'll go with option 2 here:

- [ ] add laminas-cli to `suggest` and mark the commands as `@internal` which for CLI commands implies it should not be instantiated from outside

Reply:
Sorry for the late response, but I like the solution as in laminas-cache. 👍🏻